### DISCUSSION TITLE
VillagersPlus Support

### DISCUSSION THREAD

**Discussion by gigawhip** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#40] [Polls]
[VillagersPlus](https://www.curseforge.com/minecraft/mc-mods/villagersplus-fabric) adds 4 new villagers. It is a newer mod, released just Jan 14, 2023, but has already achieved over 200k downloads. Crucially, the workbenches it adds for the new villagers are each functional, attractive, and unique blocks. This enhancement positions VillagersPlus to overtake More Villagers over time.

![image](https://user-images.githubusercontent.com/103098495/222627915-66d8b0ed-1c98-446c-982b-0acc12428c60.png)

![image](https://user-images.githubusercontent.com/103098495/222627939-05ba7fd7-10eb-43e4-be31-a127993f0e3b.png)

![image](https://user-images.githubusercontent.com/103098495/222627995-a5d2989a-96ee-445e-a280-90075f2a441e.png)

![image](https://user-images.githubusercontent.com/103098495/222628022-87d4974d-0581-44a3-b16e-8655fe8983ab.png)

**Comment by ChoiceTheorem** [comment]
This mod is already up for consideration!

**Comment by ChoiceTheorem** [comment]
Released in version 3.2.3
